### PR TITLE
improve first contact

### PR DESCRIPTION
**Pull request by tmonjalo**
- remove wrong link from GitHub templates
- fix few typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,12 @@ Starting is easy as 1-2-3
   <img src="https://contrib.rocks/image?repo=Blueforcer/awtrix-light" />
 </a>
 
+The project is built with [PlatformIO](https://platformio.org):
```
Comment (Blueforcer) [inline README.md:65]: No need to explain platformIO basics in the welcome readme. 
AWTRIX is mainly for non tech savy users. 99% of the users doesnt need and want to build the code.

File: docs/quickstart.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+Ulanzi device is switched on/off by holding left and right buttons together.
```
Comment (Blueforcer) [inline docs/quickstart.md:1]: Thats a hardware specific feature and has nothing to do with AWTIRX Light software. After unboxing you have to turn it on anyway so the user knows it before he flash awtrix light

File: README.md (hunk 1 above)
Comment (tmonjalo) [reply README.md]: It was not obvious to me, and more info does not hurt, but OK I've removed it.

File: docs/quickstart.md (hunk 2 above)
Comment (tmonjalo) [reply docs/quickstart.md]: I thought giving complete info for the supported hardware would be nice.
I've removed it.

**Pull request merged**